src/Umbraco.Web/PublishedCache/NuCache/DataSource/SerializerBase.cs

```diff
@@ -185,8 +185,16 @@ protected void WriteObject(object value, Stream stream)
             }
             else if (value is LazyCompressedString lazyCompressedString)
             {
-                PrimitiveSerializer.Char.WriteTo(PrefixCompressedStringByteArray, stream);
-                PrimitiveSerializer.Bytes.WriteTo(lazyCompressedString.GetBytes(), stream);
+                if (lazyCompressedString.Decompressed)
```

Comment:
This is problematic and shouldn't be done, I reverted this here https://github.com/umbraco/Umbraco-CMS/pull/8376/commits/57708960f0429d0e116fc06754ad45f7e370ec8d

I commented about why this was 'required' before and it shouldn't be. It would only ever become decompressed if analyzed while debugging which I've fixed here https://github.com/umbraco/Umbraco-CMS/commit/9200c46cd0b33aed42c4e66ee9e38ce6afc4d145